Sources/Model/AlbumTableViewDataSource.swift

```diff
@@ -56,7 +56,12 @@ final class AlbumTableViewDataSource : NSObject, UITableViewDataSource {
         let album = fetchResults[indexPath.section][indexPath.row]
         // Title
         cell.albumTitleLabel.text = album.localizedTitle
-        
+
```

Comment:
Didn't see any issues with the album table, but decided to add this to bring it into line with the change in PhotoCollectionViewDataSource. If a cell is re-used, and the image request is delayed, the previous/wrong image won't be shown.

Reply:
This is already being done in prepareForReuse on AlbumCell.